CHANGELOG.md

```diff
@@ -4,6 +4,171 @@ This changelog goes through the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.com).
 
+## [1.22.0](https://github.com/go-gitea/gitea/releases/tag/v1.22.0) - 2024-05-24
+
+This release includes [1478](https://github.com/go-gitea/gitea/pulls?q=is%3Apr+milestone%3A1.22.0+is%3Amerged) pull requests, which makes it the largest release in Gitea's history! It inclues a mount of new features and bug fixes.
```

Comment:
```suggestion
This release includes [1478](https://github.com/go-gitea/gitea/pulls?q=is%3Apr+milestone%3A1.22.0+is%3Amerged) pull requests, which makes it the largest release in Gitea's history! It includes a number of new features and bug fixes.
```

i am not native speaker. just a suggestion

Reply:
43f2d8b